Detectors/TRD/reconstruction/src/DataReader.cxx

```diff
@@ -33,18 +33,12 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
 {
 
   std::vector<o2::framework::ConfigParamSpec> options{
-    {"output-desc", VariantType::String, "TRDTLT", {"Output specs description string."}},
     {"verbose", VariantType::Bool, false, {"Enable verbose epn data reading."}},
-    {"verbosehalfcru", VariantType::Bool, false, {"Enable verbose for a halfcru, the halfcru contents are dumped out in hex."}},
```

Comment:
I don't see an advantage over only using the `verbose` option. It contains everything needed for debugging. It might be good at some point to add an option to only dump the full link data once in case of an error. Since this could then also be used in production. This I would add with a followup PR